sdk/typescript/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@mysten/sui.js",
   "author": "Mysten Labs <build@mystenlabs.com>",
   "description": "Sui TypeScript API(Work in Progress)",
-  "version": "0.9.0",
+  "version": "0.10.0",
```

Comment:
shouldn't this be `0.11.0` given `0.10.0` is published a month ago: https://www.npmjs.com/package/@mysten/sui.js ?

Reply:
Also should we update https://github.com/MystenLabs/sui/blob/main/sdk/typescript/CHANGELOG.md ?

Reply:
Verison: no, this brings it into alignment with what's published on `npm`.
Changelog: sure, I can update it.